### DISCUSSION TITLE
What are the words?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#497] [Galaxy Trivia] [answered]
Hidden in the grid below are eight, 7-letter words. Each word begins with the central **P** and you can move one letter in any direction to the next letter. All of the letters are used exactly once each.

![image](https://user-images.githubusercontent.com/5120290/230260907-31e0a4d2-5a52-45b2-ab34-dd5bbbccdf14.png)

**Comment by ParanoidUser** [comment]
@GameExplorer, @live627, @JustGimli, @wfrsk would you like to try? 😼

**Comment by thuongtruong1009** [comment, marked as answer]
The words hidden in the grid are:

PLENTY
PIRACY
PRESET
PRUNER
PHOBIA
POUNCE
PISTON
PRAISE
Here is the grid with

**Reply by ParanoidUser** [reply]
Not bad, but each letter must be connected to the next letter by one move in any direction. 😏

**Comment by c8f888391789da0901167ee6134c4eb2** [comment]
Are diagonal moves allowed?

**Reply by ParanoidUser** [reply]
Yup!

**Comment by GameExplorer** [comment]
Good one

**Comment by peterbaccaro** [comment]
PANCAKE
PHYSICS
PINBALL
PLASTER
POPULAR
PREMIUM
PSYCHIC
PYRAMID

**Reply by ParanoidUser** [reply]
Here are the alignments:
![image](https://user-images.githubusercontent.com/5120290/230395841-4bde5de8-73ba-4182-ad39-516ca2f37e38.png)
